src/style.css

```diff
@@ -1 +1,29 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
Be consistent with your margins. It's a good practice to add margin on the top or bottom only. This approach helps to prevent issues with margin collapsing and makes the margin behavior more predictable.